### ISSUE TITLE
ExtendsContentIntoTitleBar results in incorrectly sized TitleBar buttons

### ISSUE THREAD

**Issue by YexuanXiao** [microsoft/microsoft-ui-xaml#8393]
### Describe the bug

The buttons have the right height, the wrong width, and tiny icons.

### Steps to reproduce the bug

```c#
public MainWindow()
{
    this.InitializeComponent();

    // Hide default title bar.
    ExtendsContentIntoTitleBar = true;
    SetTitleBar(TitleBar);
}
```

### Screenshots

![image](https://user-images.githubusercontent.com/20025591/232096241-3ba7f348-b319-4737-b58b-7da37f4f852d.png)
![image](https://user-images.githubusercontent.com/20025591/232096270-006c7b09-5c6a-464a-98d2-5368ede453bb.png)

### NuGet package version

WinUI 3 - Windows App SDK 1.2.5: 1.2.230313.1
WinUI 3 - Windows App SDK 1.3: 1.3.230331000

### Windows version

Windows Insider Build (23430, 25336)

**Comment by l619534951** [comment]
don't use that, use this

```c#
AppWindow.TitleBar.ExtendsContentIntoTitleBar = true
```

**Comment by Dealman** [comment]
> don't use that, use this
> 
> ```cs
> AppWindow.TitleBar.ExtendsContentIntoTitleBar = true
> ```

This works, but for me the default backgrounds are black which looks weird - but is easily resolved;
```cs
AppWindow.TitleBar.ButtonBackgroundColor = Microsoft.UI.Colors.Transparent;
AppWindow.TitleBar.ButtonInactiveBackgroundColor = Microsoft.UI.Colors.Transparent;
```

**Comment by pratikone** [comment]
fixed with winappsdk 1.4 where appwindow titlebar is being used under the hood by winui 3 custom titlebar
